airbyte-webapp/src/pages/ConnectionPage/pages/AllConnectionsPage/AllConnectionsPage.tsx

```diff
@@ -41,11 +39,7 @@ const AllConnectionsPage: React.FC = () => {
           <ConnectionsTable connections={connections} />
         </MainPageWithScroll>
       ) : (
-        <EmptyResourceListView
-          resourceType="connections"
-          onCreateClick={onCreateClick}
-          disableCreateButton={!allowCreateConnection}
-        />
+        <EmptyResourceListView resourceType="connections" onCreateClick={onCreateClick} />
```

Comment:
You can remove `disableCreateButton` from this component now.

Reply:
Fixed